x-pack/test/functional/es_archives/spaces/disabled_features/data.json

```diff
@@ -1,40 +1,45 @@
 {
   "type": "doc",
   "value": {
-    "index": ".kibana",
-    "type": "doc",
     "id": "space:default",
+    "index": ".kibana_1",
```

Comment:
Is this esArchive still used?

Reply:
See https://github.com/elastic/kibana/pull/102533#issuecomment-883246188

Reply:
Ah, I see. Works for me.